layers/+completion/auto-completion/funcs.el

```diff
@@ -93,7 +93,14 @@ Available PROPS:
                  ,(format "Company backend list for %S." mode)) result)
         ;; add backends
         (dolist (backend backends)
-          (push `(add-to-list ',raw-backends-var-name ',backend) result))
+          ;; add tabnine backend if enabled
```

Comment:
This looks overly complicated. Would this work instead?
```elisp
(dolist (backend backends)
  (push `(add-to-list ',raw-backends-var-name ',backend) result))
(when auto-completion-enable-tabnine
  (push `(add-to-list ',raw-backends-var-name 'company-tabnine) result))
```
(Mind you I'm pretty unfamiliar with elisp.)